NSS-Day-Cohort-55/tabloidcli-lyrical-lorikeets
# Ls 5 search all

### Desc
Added functionality for Search All for Tag
### Testing Instructions
Select Vampire (2) color scheme (optional)
Search (6)
Search All (4)
Search for Tag, eg: 'nerdy', results should populate
Search for tag that doesn't exist, not found msg should appear
